docs/eventlisteners.md

```diff
@@ -239,18 +323,17 @@ spec:
 
 ### CEL Interceptors
 
-CEL interceptors parse expressions to filter requests based on JSON bodies and
+CEL Interceptors parse expressions to filter requests based on JSON bodies and
 request headers, using the [CEL](https://github.com/google/cel-go) expression
 language. Please read the
 [cel-spec language definition](https://github.com/google/cel-spec/blob/master/doc/langdef.md)
 for more details on the expression language syntax.
 
 In addition to the standard
 [CEL expression language syntax](https://github.com/google/cel-spec/blob/master/doc/langdef.md),
-additional supported features include case-insensitive matching on request
-headers.
+additional supported features described [CEL expressions](./cel_expressions.md)
```

Comment:
missing period `[CEL expressions](./cel_expressions.md).`

I also noticed that this sentence is a little confusing `In addition to the standard CEL expression language syntax, additional supported features described CEL expressions`. Maybe I'm misreading it, but I think that it sounds confusing.

Maybe it should be something like `Along with the standard CEL expression language syntax, Triggers supports these additional CEL expressions.` ?

Reply:
good catch :)